src/sentry/api/serializers/models/integration.py

```diff
@@ -197,3 +205,15 @@ def serialize(self, obj, attrs, user):
         data = super().serialize(obj, attrs, user)
         data["externalIssues"] = attrs.get("external_issues", [])
         return data
+
+
+@register(ExternalTeam)
+class ExternalTeamSerializer(Serializer):
+    def serialize(self, obj, attrs, user):
+        provider = EXTERNAL_PROVIDERS.get(ExternalProviders(obj.provider), "unknown")
```

Comment:
I think we'll see this int -> string a lot. Can you make this line into a function that we can reuse?

Reply:
@mgaeta I thought I should wait until I needed to reuse the logic, since its just a one-liner. But I can do that!